frontend/src/pages/modelRegistry/screens/ModelRegistry.tsx

```diff
@@ -1,26 +1,28 @@
 import React from 'react';
 import ApplicationsPage from '~/pages/ApplicationsPage';
-import { ModelRegistryContext } from '~/concepts/modelRegistry/context/ModelRegistryContext';
 import useRegisteredModels from '~/concepts/modelRegistry/apiHooks/useRegisteredModels';
+import { ModelRegistrySelectorContext } from '~/concepts/modelRegistry/context/ModelRegistrySelectorContext';
+import TitleWithIcon from '~/concepts/design/TitleWithIcon';
+import { ProjectObjectType } from '~/concepts/design/utils';
 import EmptyRegisteredModels from './EmptyRegisteredModels';
-import RegisteredModelsTableToolbar from './RegisteredModelsTableToolbar';
 import RegisteredModelListView from './RegisteredModelListView';
+import ModelRegistrySelector from './ModelRegistrySelector';
 
 const ModelRegistry: React.FC = () => {
-  const { preferredModelRegistry } = React.useContext(ModelRegistryContext);
+  const { preferredModelRegistry } = React.useContext(ModelRegistrySelectorContext);
   const [registeredModels, loaded, loadError] = useRegisteredModels();
 
   return (
     <ApplicationsPage
       empty={registeredModels.size === 0}
       emptyStatePage={
-        <>
-          <RegisteredModelsTableToolbar />
-          <EmptyRegisteredModels preferredModelRegistry={preferredModelRegistry?.metadata.name} />
-        </>
+        <EmptyRegisteredModels preferredModelRegistry={preferredModelRegistry?.metadata.name} />
+      }
+      title={
+        <TitleWithIcon title="Deployed models" objectType={ProjectObjectType.deployedModels} />
       }
-      title="Registered models"
       description="View and manage your registered models."
+      headerContent={<ModelRegistrySelector />}
       loadError={loadError}
       loaded={loaded}
       provideChildrenPadding
```

Comment:
can you add `removeChildrenTopPadding`  to remove extra top padding?

with padding
<img width="400px" src="https://github.com/opendatahub-io/odh-dashboard/assets/97534722/de9e99c0-1fa5-4b44-82bf-2fed5a89cd84" />

without padding
<img width="400px" src="https://github.com/opendatahub-io/odh-dashboard/assets/97534722/2e113998-64ef-45cf-be6e-6065fc67c5bd" />

Reply:
Great suggestion!